.yetus/excludes.txt

```diff
@@ -1,10 +1,12 @@
 ^tests/integration/config.yml
+^tests/eden/eclient/testdata/maridb.txt
 ^pkg/mkimage-raw-efi/initramfs-init
 ^pkg/mkimage-raw-efi/nlplug-findfs.c
 ^pkg/pillar/rootfs/fscrypt.conf
 ^pkg/fw/
 ^pkg/debug/abuild/
 ^pkg/pillar/zedpac/
+^pkg/kvm-tools/
```

Comment:
But we have code in pkg/kvm-tools (like the Dockerfile etc) which we do want to check.
Can't you exclude something more specific?

Reply:
Indeed, we can exclude only pkg/kvm-tools/include and pkg/kvm-tools/dtrace... will update the PR...